src/deployment/cd.md

```diff
@@ -228,6 +228,9 @@ Create a file at `ios/ci_scripts/ci_post_clone.sh` and add the content below.
 ```sh
 #!/bin/sh
 
+# Fail this script if any subcommand fails.
```

Comment:
This snippet is copied as a code excerpt from the https://github.com/flutter/website/blob/main/examples/deployment/xcode_cloud/ci_post_clone.sh file. Could you add this new command there as well?

Reply:
Done. Thank you.